.github/workflows/ci-pypi-publish.yml

```
@@ -0,0 +1,71 @@
+name: pyp publish
+
+on:
+  workflow_call:
+    inputs:
+      python-version:
+        description: 'Python version to use'
+        required: false
+        type: string
+        default: '3.15'
```

Can use 3.x for latest stable version:
```suggestion
        default: '3.x'
```